### PR TITLE
Change the title of the Keycloak Truststore guide to make the intent more clear

### LINKED ISSUE #14960: Change the title of the Keycloak Truststore guide to make the intent more clear
### Discussed in https://github.com/keycloak/keycloak/discussions/12310

<div type='discussions-op-text'>

<sup>Originally posted by **dasniko** June  2, 2022</sup>
Hi,
In the Keycloak Guides, there are two different options mentioned for configuring the truststore, which is used on outgoing https requests:

1. [Configuring TLS / Using a truststore](https://www.keycloak.org/server/enabletls#_using_a_truststore)
This is using the `https-trust-store...` config, here is mentioned _"This truststore is used for outgoing https requests as well as mTLS requests."_
2. [Configuring outgoing HTTP requests](https://www.keycloak.org/server/outgoinghttp#_outgoing_https_request_truststore)
This guide mentions the `spi-truststore-file-...` config for outgoing https requests.

Which is correct?
It seems that only the latter is working properly with an `ldaps://` connection.
In our project, we have an extension using the `SimpleHttp` class, but outgoing https requests fail with problems using the cert path. The logs say that no proper cert can be found, but it is actually in the truststore we've configured with the `spi-truststore-file...` aproach.

We can't use the `https-trust-store...` config approach, as it fails to start Keycloak when having a custom keystore configured at the same time (with `https-key-store-file`).

Is anybody able to explain for which approach I do have to use which config approach?
And does the `SimpleHttp` class really use the truststore configured via the SPI config approach?
I'm a bit lost and don't know where to look else to get it work.</div>

### PR DESCRIPTION
**Pull request by pedroigor**
Closes #14960

### CODE REVIEW

File: docs/guides/src/main/server/keycloak-truststore.adoc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <#import "/templates/kc.adoc" as kc>
 
 <@tmpl.guide
-title="Configuring a Truststore"
+title="Configuring a Truststore for Outgoing TLS connections"
```
Comment (stianst) [inline docs/guides/src/main/server/keycloak-truststore.adoc:5]: I don't think this is necessary as it's explained in the summary, and would make the title very long/ugly

File: docs/guides/src/main/server/outgoinghttp.adoc [hunk 2]
```diff
@@ -118,7 +118,7 @@ In this example, the following occurs:
 * The special value NO_PROXY for the proxy-uri is used, which means that no proxy is used for hosts matching the associated hostname pattern.
 * A catch-all pattern ends the proxy-mappings, providing a default proxy for all outgoing requests.
 
-== Outgoing HTTPS request truststore
+== Enabling TLS connections
```
Comment (stianst) [inline docs/guides/src/main/server/outgoinghttp.adoc:121]: TLS connections are already enabled though? This is about adding additional certs that you trust to the store.

Comment (stianst) [reply docs/guides/src/main/server/outgoinghttp.adoc:121]: On that note it would also be nice to have a non-Java way of trusting more certs ;)

Comment (stianst) [reply docs/guides/src/main/server/outgoinghttp.adoc:121]: By the way there's this proposal from the community:
https://github.com/keycloak/keycloak-community/pull/345

Haven't looked at it yet though.

File: docs/guides/src/main/server/keycloak-truststore.adoc (hunk 1 above)
Comment (pedroigor) [reply docs/guides/src/main/server/keycloak-truststore.adoc:5]: The summary already explains the purpose of configuring a trust store. The proposed change is basically making it more explicit when reading the title.

The problem here is that by reading `Configuring a Truststore` is not clear at all the intent of the guide. That is why I'm changing the title.

Comment (stianst) [reply docs/guides/src/main/server/keycloak-truststore.adoc:5]: What about:

```suggestion
title="Configuring trusted certificates for outgoing requests"
```

**Review by vmuzikar — APPROVED** [review]

**Pull request merged**